apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/mapAssertions.kt

```diff
@@ -11,6 +11,8 @@ import ch.tutteli.kbox.identity
  * Starts a sophisticated `contains` assertion building process based on this [Expect].
  *
  * @return The newly created builder.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.deprecated.MapAssertionSamples.containsFeature
```

Comment:
Can you please rename this to containsBuilder (to be consistent with CharSequenceAssertionSamples.containsBuilder)